cluster/network.go

```diff
@@ -75,90 +75,113 @@ const (
+	AciMTUHeadRoom                          = "aci_mtu_head_room"
+	AciNodePodIfEnable                      = "aci_node_pod_if_enable"
+	AciSriovEnable                          = "aci_sriov_enable"
+	AciMultusDisable                        = "aci_multus_disable"
+	AciNoWaitForServiceEpReadiness          = "aci_no_wait_for_service_ep_readiness"
+	AciAddExternalSubnetsToRdconfig         = "aci_add_external_subnets_to_rdconfig"
+	AciServiceGraphEndpointAddDelay         = "aci_service_graph_endpoint_add_delay"
+	AciHppOptimization                      = "aci_hpp_optimization"
+	AciSleepTimeSnatGlobalInfoSync          = "aci_sleep_time_snat_global_info_sync"
+	AciOpflexAgentOpflexAsyncjsonEnabled    = "aci_opflex_agent_opflex_asyncjson_enabled"
+	AciOpflexAgentOvsAsyncjsonEnabled       = "aci_opflex_agent_ovs_asyncjson_enabled"
+	AciOpflexAgentPolicyRetryDelayTimer     = "aci_opflex_agent_policy_retry_delay_timer"
+	AciAciMultipod                          = "aci_aci_multipod"
+	AciOpflexDeviceReconnectWaitTimeout     = "aci_opflex_device_reconnect_wait_timeout"
+	AciAciMultipodUbuntu                    = "aci_aci_multipod_ubuntu"
+	AciDhcpRenewMaxRetryCount               = "aci_dhcp_renew_max_retry_count"
+	AciDhcpDelay                            = "aci_dhcp_delay"
+	AciUseSystemNodePriorityClass           = "aci_use_system_node_priority_class"
+	AciAccProvisionOperatorMemoryRequest    = "aci_acc_provision_operator_memory_request"
+	AciAccProvisionOperatorMemoryLimit      = "aci_acc_provision_operator_memory_limit"
+	AciAciContainersControllerMemoryRequest = "aci_aci_containers_controller_memory_request"
+	AciAciContainersControllerMemoryLimit   = "aci_aci_containers_controller_memory_limit"
+	AciAciContainersHostMemoryRequest       = "aci_aci_containers_host_memory_request"
+	AciAciContainersHostMemoryLimit         = "aci_aci_containers_host_memory_limit"
+	AciAciContainersOperatorMemoryRequest   = "aci_aci_containers_operator_memory_request"
+	AciAciContainersOperatorMemoryLimit     = "aci_aci_containers_operator_memory_limit"
+	AciMcastDaemonMemoryRequest             = "aci_mcast_daemon_memory_request"
+	AciMcastDaemonMemoryLimit               = "aci_mcast_daemon_memory_limit"
+	AciOpflexAgentMemoryRequest             = "aci_opflex_agent_memory_request"
+	AciOpflexAgentMemoryLimit               = "aci_opflex_agent_memory_limit"
+	AciAciContainersMemoryRequest           = "aci_aci_containers_memory_request"
+	AciAciContainersMemoryLimit             = "aci_aci_containers_memory_limit"
+	AciUseDigest                            = "aci_use_digest"
+	AciCnideployVersion                     = "aci_cnideploy_version"
+	AciAciContainersHostVersion             = "aci_aci_containers_host_version"
+	AciOpflexAgentVersion                   = "aci_opflex_agent_version"
+	AciOpenvswitchVersion                   = "aci_openvswitch_version"
+	AciAciContainersControllerVersion       = "aci_aci_containers_controller_version"
```

Comment:
Is this intentional to have these 6 options be exclusive to RKE 1.4.x? Don't see them in 1.3.x PR - https://github.com/rancher/rke/pull/3349/files.

Let us know what's the reason for it as by default I would expect field additional to match between 1.3.x and 1.4.x.

Reply:
Thanks for pointing it out. I'd to remove those.
Updated it.